storage/oasis/nodeapi/api.go

```diff
@@ -206,10 +206,54 @@ type Proposal governance.Proposal
 // ....................  Runtimes  ....................
 // ....................................................
 
+// FallibleResponse stores the response to a node query. If the call was successful,
```

Comment:
This file is already kinda long and might need to eventually be split. But for now, I've been trying to at least keep it limited to the top-level types of the node API to keep it easily navigable. Can you please move the newly added lines into a new file, e.g. `evm.go`?